src/modules/fancyzones/lib/fancyzones.rc

```diff
@@ -7,6 +7,7 @@ STRINGTABLE
 BEGIN
     IDS_SETTING_DESCRIPTION                                    "Create window layouts to help make multi-tasking easy"
     IDS_SETTING_DESCRIPTION_SHIFTDRAG                          "On: Hold Shift key or any non-primary mouse button to enable zones while dragging"
+    IDS_SETTING_DESCRIPTION_MOUSESWITCH                        "Use a non-primary mouse button to activate zones while dragging"
```

Comment:
Wording shoud probably be more similar to what we have in IDS_SETTING_DESCRIPTION_SHIFTDRAG, to emphasize that they do the same thing.

Reply:
We need to update `On: Hold Shift key or any non-primary mouse button to enable zones while dragging` because with this fix it changes the behavior, and since we are there let's remove `On:`
```
"Hold Shift key to enable zones while dragging"
```

Reply:
`"Use a non-primary mouse button to activate zones while dragging"`
change it to:
```
"Click (and release) a non-primary mouse button to activate zones while dragging"

```

Reply:
I've changed `enable` to `activate`

Reply:
I deleted my previous comments because I was suggesting descriptions that apply to the previous PR. Let me rephrase:
`"On: Hold Shift key or any non-primary mouse button to enable zones while dragging"` -> `"Hold Shift key to activate zones while dragging"`

`"Use a non-primary mouse button to activate zones while dragging"` ->
`"Hold  a non-primary mouse button to activate zones while dragging"`